multi_cgra/parser/test/conftest.py

```diff
@@ -0,0 +1,11 @@
+import os
+"""
+    Use `pytest Parser_test.py --arch_file /path/to/arch/file` to execute the test case of parser.
```

Comment:
Can `pytest Parser_test.py --arch_file arch.yaml` just be added into our Github Action Workflow.